app/src/main/java/io/homeassistant/companion/android/webview/WebViewActivity.kt

```diff
@@ -483,4 +579,36 @@ class WebViewActivity : AppCompatActivity(), io.homeassistant.companion.android.
             }
         }, 5000)
     }
+
+    private fun setupPanelShortcuts() {
+        waitForConnection()
```

Comment:
This isn't doing what you think it is.  This doesn't halt execution and wait for a connection.  I think you are better off calling this in the external bus, connection-status block without the `waitForConnection()`.